content/posts/2020-02-19-sagas-typing/index.mdx

```diff
@@ -0,0 +1,287 @@
+---
+title: How to type a saga
```

Comment:
Think about SEO when writing your title. I would say that you need "typescript", "redux-saga" in the words of the title